ansible/roles/ocp4-workload-debezium-demo/tasks/provision_mysql_demo.yaml

```diff
@@ -33,18 +33,13 @@
     resource_definition: "{{ lookup('template', 'mysql.yaml.j2') }}"
 
 - name: Wait for Kafka Cluster to start
-  k8s_facts:
+  k8s_info:
     kind: Kafka
     name: demo
     namespace: '{{ ocp4_workload_debezium_demo_mysql_namespace }}'
     api_version: kafka.strimzi.io/v1beta1
-  register: kafka_resource
-  until: >-
-    (kafka_resource.resources is defined) 
-    and (kafka_resource.resources|length>0)
-    and (kafka_resource.resources[0].status is defined) 
-    and (kafka_resource.resources[0].status.conditions is defined)
-    and (kafka_resource.resources[0].status.conditions[0].type == 'Ready') 
+  register: r_k8s_info
+  until: r_k8s_info.resources[0].status.conditions[0].type | default('') == 'Ready'
```

Comment:
Same as previous comment.

Reply:
Same story here. It isn't a problem with ansible 2.9 and later.